### PR TITLE
Add autoplay to carousel and update schema

### PR DESCRIPTION
**Pull request by vinothvino42**
In this PR, I added autoplay and duration to the carousel widget

### CODE REVIEW

File: lib/widget/carousel.dart [hunk 1]
```diff
@@ -40,6 +40,9 @@ class Carousel extends StatefulWidget
           _controller.layout = CarouselLayout.values.from(input),
       'autoLayoutBreakpoint': (value) =>
           _controller.autoLayoutBreakpoint = Utils.optionalInt(value, min: 0),
+      'autoplay': (value) => _controller.autoplay = Utils.optionalBool(value),
+      'autoplayInterval': (value) =>
```
Comment (vusters) [inline lib/widget/carousel.dart:44]: add min: 1

File: assets/schema/ensemble_schema.json [hunk 2]
```diff
@@ -1766,6 +1766,14 @@
                     "indicatorHeight": {
                       "type": "integer"
                     },
+                    "autoplay": {
```
Comment (vusters) [inline assets/schema/ensemble_schema.json:1769]: default is false according to your code. Default=false is what we want, just just change the schema here.

Comment (vinothvino42) [reply assets/schema/ensemble_schema.json:1769]: @vusters Can you check the new changes? I updated the two things (min value and default false changes) which you mentioned.

File: lib/widget/carousel.dart (hunk 1 above)
Comment (vusters) [reply lib/widget/carousel.dart:44]: yes, since you use seconds, you want the user to enter at least 1 sec.

**Review by vusters — APPROVED** [review]
Ship it

**Pull request merged**